app/workers/work_packages/progress/job.rb

```diff
@@ -39,4 +39,8 @@ class WorkPackages::Progress::Job < ApplicationJob
     perform_limit: 1,
     key: -> { "WorkPackagesProgressJob" }
   )
+
+  retry_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError,
+           wait: 5.minutes,
+           attempts: 3
```

Comment:
I wouldn't want to loose the data change that the job carries out. If anything, I would set the `retry_on` for this job to `:unlimited` and [overwrite the default of 5](https://api.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on).

Reply:
And keep the 5 minute interval retry?